lib/jira/resource/agile.rb

```diff
@@ -15,19 +15,20 @@ def self.all(client)
 
       def self.get_backlog_issues(client, board_id, options = {})
         options[:maxResults] ||= 100
-        response = client.get("/rest/agile/1.0/board/#{board_id}/backlog?maxResults=#{options[:maxResults]}")
+        response = client.get(path_base(client) + "board/#{board_id}/backlog", options)
         parse_json(response.body)
       end
 
       def self.get_sprints(client, board_id, options = {})
         options[:maxResults] ||= 100
-        response = client.get("/rest/agile/1.0/board/#{board_id}/sprint?maxResults=#{options[:maxResults]}")
+        response = client.get(path_base(client) + "/board/#{board_id}/sprint?#{hash_to_query_string(options)}")
         parse_json(response.body)
       end
 
       def self.get_sprint_issues(client, sprint_id, options = {})
         options[:maxResults] ||= 100
-        response = client.get("/rest/agile/1.0/sprint/#{sprint_id}/issue?maxResults=#{options[:maxResults]}")
+        response = client.get(path_base(client) +
+                                  "rest/agile/1.0/sprint/#{sprint_id}/issue?maxResults=#{options[:maxResults]}")
```

Comment:
Here you forgot to remove `rest/agile/1.0/` from the URL when added `path_base(client)` 😉